### ISSUE TITLE
Move random button to top of page, and add shuffle button in corner of games for ease of use

### ISSUE THREAD

**Issue by TheDonk113** [Wade7wastaken/TrulyUnblockedGames#5]

**Comment by TheDonk113** [comment]
If shuffle button is impossible so be it I guess

**Comment by Wade7wastaken** [comment]
I agree that the random button should be moved to the top of the page, but what would a shuffle button do? Doesn't the random button do the same thing? A shuffle button wouldn't be impossible, but I don't see why it needs to be added.

**Comment by TheDonk113** [comment]
Shuffle button would be nice so that you can switch to another game without having to go to the website home page

**Comment by Wade7wastaken** [comment]
Yeah ok, but tell me what it would do and where it would go.

**Comment by TheDonk113** [comment]
in one of the corners, it switches to another random game. That way you wouldnt have to go back to the main page to switch games. Could also go in middle around the edge idk.

**Comment by Wade7wastaken** [comment]
Once you click on a game link, I can’t control what’s on the page anymore. You’ll just have to deal with going back the main page.
